packages/nimble-components/CONTRIBUTING.md

```diff
@@ -81,6 +91,8 @@ Create a new folder named after your component with some core files:
 | index.ts                               | Contains the component class definition and registration. All TypeScript logic contained in the component belongs here.                                                                                                                                                    |
 | styles.ts                              | Contains the styles relevant to this component. Note: Style property values that can be shared across components belong in [theme-provider/design-tokens.ts](/packages/nimble-components/src/theme-provider/design-tokens.ts).                                             |
 | template.ts                            | Contains the template definition for components that don't use a fast-foundation template.                                                                                                                                                                                 |
+| types.ts                               | Contains any enum-like types defined by the component                                                                                                                                                                                                                      |
+| models/                                | A folder containing any classes or interfaces that are part of the component API or implementation                                                                                                                                                                         |
```

Comment:
Looks like wafer-map used the term `modules` instead, but I think I like models more. Each JS file is a module so I guess the term is technically not wrong but model for these files does seem a bit better I guess.

https://github.com/ni/nimble/tree/main/packages/nimble-components/src/wafer-map/modules

Reply:
In the wafer map case, a lot of the code isn't purely data structures/types, it's data processing logic (which was formerly contained in Angular services). While I like `models` for data structure code, my initial reaction is that `models` wouldn't be right for that code that has so much logic. I don't like `modules` much either because it's so general, but haven't thought of a more precise term that I like enough to suggest.

So I'd propose we leave these docs as-is (sounds like you agree) but don't push back on the wafer map naming yet.